### DISCUSSION TITLE
Modifier key for dragging windows

### DISCUSSION THREAD

**Discussion by pharwood** [rxhanson/Rectangle#320] [Ideas]
Would it be possible to add the option to only resize windows when dragging if a modifier key was being pressed? Similar to the option for Desktop & Screen Saver hot corners in System Preferences.

**Comment by rxhanson** [comment]
This is a good idea. It might take me a little while to get around to adding it in, though (PRs welcome). 

Which modifier key(s) would you use? I ask because the shortcut recorder in Rectangle by default doesn't have a mode for modifier key only recording. I don't plan on modifying the shortcut recorder in Rectangle at this time, so the modifier key(s) would have to be hard coded or adjusted via terminal commands at first cut.

**Comment by pharwood** [comment]
Hi Ryan,
Glad you like the idea. Command, option or the control key would be ideal for the modifier key. Command would be my personal preference as it is usually right under my left thumb.

Best regards
Paul

**Reply by pharwood** [reply]
Thanks for implementing this in the latest release. Works a treat.

**Reply by rxhanson** [reply]
Thanks for checking it out and letting me know!
